tests/cpp17/generated_cpp17/monster_test_generated.h

```diff
@@ -14,6 +14,13 @@ static_assert(FLATBUFFERS_VERSION_MAJOR == 2 &&
               FLATBUFFERS_VERSION_REVISION == 6,
              "Non-compatible flatbuffers version included");
 
+static inline bool VerifyNestedFlexBuffer(const flatbuffers::Vector<uint8_t> *nv,
```

Comment:
how is this in the `flexbuffers` namespace?

What happens if two files with this function in it get included? I think that's still a compiler error.
If that is so, you can either:
* stick an `#ifdef`` around this function much like include guards
* or actually stick it in a `flexflatutils.h` or similar, a file only to be included after both are used (maybe better)
* or (for completeness), you could generate this function specific to the field name it is used for
* or (for completeness) do the body of this code entirely inline (which would call `flex()` multiple times which is slower.

the 2nd one maybe best?

Reply:
Yeah, #2 sounds the best actually. Then I can just include that file in the generated output if needed.